app/Main.hs

```diff
@@ -74,34 +85,47 @@ newtype PostResponse = PostResponse Int
 instance ToJSON PostResponse where
   toJSON (PostResponse x) = object [ "userID" .= x ]
 
+data UserRow = UserRow
+  { username :: Text
```

Comment:
Same idea as the other `User` types. This is the only one that doesn't use `Username`.